app/assets/stylesheets/editable/bootstrap-editable.scss

```diff
@@ -0,0 +1,663 @@
+/*! X-editable - v1.5.1 
+* In-place editing with Twitter Bootstrap, jQuery UI or pure jQuery
+* http://github.com/vitalets/x-editable
+* Copyright (c) 2013 Vitaliy Potapov; Licensed MIT */
+.editableform {
+    margin-bottom: 0; /* overwrites bootstrap margin */
+}
+
+.editableform .control-group {
+    margin-bottom: 0; /* overwrites bootstrap margin */
```

Comment:
Use `//` comments everywhere<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered line-height, margin-bottom, white-space